samples/Esprima.Benchmark/AstTreeWalkBenchmark.cs

```diff
@@ -89,7 +89,7 @@ public void Visit(Node node)
                         }
                     }
 
-                    if (childNode.ChildNodes.Count > 0)
+                    if (!childNode.ChildNodes.IsEmpty())
```

Comment:
Yeah, I hesitated over this because most of the time `IsEmpty` is just a few method calls wrapped in a try-finally block. So it could as well be declared as a property. However, `ChildNodes` provide a way for end users to implement child nodes enumeration with yield return and in such cases there will be an allocation. Thus, I finally decided to keep it a method.